test/e2e-smoke-tests.sh

```diff
@@ -35,7 +35,7 @@ function knative_setup() {
 
 # Script entry point.
 
-initialize $@
+initialize $@ --skip-istio-addon
```

Comment:
On line 33 we're not installing Istio, thus by doing this I'm pretty sure the test will fail in the setup phase.

Reply:
Also, the other tests pass `--skip-istio-addon` because we install Istio ourselves in the test setup. This test is intended to exercise the case where you install Knative from the serving manifest, as instructed in the docs.